src/components/CommentCard.jsx

```diff
@@ -0,0 +1,8 @@
+const CommentCard = ({ comment }) => {
+    return <li className="comment-card">
+        <p>{comment.body}</p>
+        <p className="comment-author">{comment.author}</p>
```

Comment:
I think it would be good to also include the time the comment was posted as well in the comment card